src/kata6.joinNames.js

```diff
@@ -1,3 +1,14 @@
-const joinNames = namesObj => {};
+/* eslint-disable prettier/prettier */
+const joinNames = namesObj => {
+   if (namesObj.length >= 2) {
+    const newArray = namesObj.map(x=> x.name);
+    const newString = `${newArray[newArray.length-2]} & ${newArray[newArray.length-1]}`;
+       newArray.splice(newArray.length-2);
+       newArray.push(newString);
+       return newArray.join(', ');
+    } else {const newArray = namesObj.map(x=> x.name); return newArray.join(', ');}
```

Comment:
Nice solution! It could be made more readable by using the linter though, plus renaming some of the variables to make it clearer what they are:
```suggestion
  if (namesObj.length >= 2) {
    const namesArray = namesObj.map(x => x.name);
    const finalPair = `${namesArray[namesArray.length - 2]} & ${
      namesArray[namesArray.length - 1]
    }`;

    namesArray.splice(namesArray.length - 2);
    namesArray.push(finalPair);
    return namesArray.join(", ");
  }
  const namesArray = namesObj.map(x => x.name);
  return namesArray.join(", ");
```